<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>15 Numbers Puzzle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backgournd:rgb(170, 170, 255)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>467</width>
      <height>314</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.445, fy:0.477273, stop:0 rgba(94, 128, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QPushButton" name="button_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>31</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>105</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>31</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_3">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>31</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_4">
     <property name="geometry">
      <rect>
       <x>349</x>
       <y>31</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_9">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>167</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_10">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>167</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_11">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>167</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>11</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_12">
     <property name="geometry">
      <rect>
       <x>349</x>
       <y>167</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_13">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>235</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>13</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_14">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>235</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>14</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_15">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>235</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_16">
     <property name="geometry">
      <rect>
       <x>349</x>
       <y>235</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_7">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>99</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_5">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>99</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_6">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>99</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_8">
     <property name="geometry">
      <rect>
       <x>349</x>
       <y>99</y>
       <width>100</width>
       <height>65</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(184, 21, 45);
color:rgb(238, 192, 24);</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>225</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Shuffle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>225</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>227</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Century Schoolbook&quot;;
</string>
       </property>
       <property name="text">
        <string>Moves:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Century Schoolbook&quot;;
color:rgb(170, 85, 255)</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>statusBar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
